src/styles/tools/_mixin.scss

```diff
@@ -0,0 +1,4 @@
+@mixin square($star-size) {
```

Comment:
I write in logo u have also a square 40x40px. use your mixin with that logo. keep in mind that here mixin take some value from @include declaration. so here in mixin I can type @mixin square($x) or @mixin square($y) but include declare what I will take. so $x or $y = $value-from-include. rework your mixin that it also work for logo